### PR TITLE
Correct email links to use mailto

### PR DESCRIPTION
**Pull request by altendky**
As is I get links like `https://www.chia.net/careers/jobs@chia.net`.  I have not had a chance to test this and I haven't worked with GitHub pages much yet.

### CODE REVIEW

**Comment by altendky** [comment]
Hmm, I should probably look at this more to figure out the proper link format here...

**Comment by altendky** [comment]
I just ran jekyll and tested these three links locally and they work.

File: _data/strings.yml [hunk 1]
```diff
@@ -90,8 +90,7 @@ en: # English
 
     When you are authorized to use the Chia® brand and respect the Chia® brand guidelines, you help us ensure the strength of the brand, while protecting the entire community from being misled, confused or mistreated.
 
-    Please read over the following guidelines carefully, and never hesitate to reach out with questions or permission requests at hello@chia.net.
-
+    Please read over the following guidelines carefully, and never hesitate to reach out with questions or permission requests at [hello@chia.net](mailto:hello@chia.net).
     Happy Farming!
```
Comment (altendky) [inline _data/strings.yml:94]: Hmm, somehow this removes the blank line between the email address line and the 'Happy Farming!'.

Comment (altendky) [reply _data/strings.yml:94]: Yes, the blank line is deleted.  Recovering now.

**Review by hoffmang9 — APPROVED** [review]
AOK

**Pull request merged**